homeassistant/components/knx/light.py

```diff
@@ -63,8 +66,15 @@ async def async_setup_entry(
 ) -> None:
     """Set up light(s) for KNX platform."""
     knx_module: KNXModule = hass.data[DOMAIN]
+    platform = async_get_current_platform()
+    platform_controller = KnxUiEntityPlatformController(
+        knx_module=knx_module,
+        entity_platform=platform,
+        entity_class=KnxUiLight,
+    )
+    knx_module.config_store.platform_controllers[Platform.LIGHT] = platform_controller
```

Comment:
Instead of updating this attribute directly, I'd add a method to the config store called eg `add_platform` that takes parameters `Platform` and `PlatformControllerBase`. Then use that method here and in other platforms as needed.